packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,96 @@
+:where(.hop-Popover *) {
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
+    color: var(--color);
+
+    outline: none;
+}
+
+.hop-Popover__dialog {
```

Comment:
shouldn't all these variables be set directly on .hop-Popover?

Reply:
because `HopperProvider` injects the tokens into a sub-element of `.hop-Popover`, which is why the variables are defined inside the dialog. 

<img width="585" alt="Capture d’écran, le 2024-06-04 à 13 15 41" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/d8f6bc0d-5721-4b7b-bd80-2c8879b1fecb">

Reply:
oh I see. I feel like I may have seen an issue you were having before and adding hopper provider may have been a fix to something, but I will ask anyway, why is HopperProvider added in your component?
This is for my own knowledge. :P

Reply:
Well, as the popover is generated outside the DOM via portal, the component elements don't have access to the variable. To solve this problem, we need to add the Provider inside 

<img width="550" alt="Capture d’écran, le 2024-06-04 à 16 28 22" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/ae9b1808-9a85-4074-a2ff-6e84f3e41a3b">